packages/vue-output-target/rollup.config.mjs

```diff
@@ -1,4 +1,4 @@
-import pkg from './package.json';
+import pkg from './package.json' assert { type: 'json' };
```

Comment:
Can we use `with` here? `assert` is deprecated per https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules